src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesServiceTest.kt

```diff
@@ -513,19 +517,21 @@ internal class ActivitiesServiceTest {
       )
       whenever(nomisApiService.createAllocation(any(), any())).thenThrow(RuntimeException("test"))
 
-      assertThatThrownBy {
-        activitiesService.createAllocation(
-          AllocationDomainEvent(
-            eventType = "dummy",
-            version = "1.0",
-            description = "description",
-            occurredAt = LocalDateTime.now(),
-            additionalInformation = AllocationAdditionalInformation(
-              allocationId = ALLOCATION_ID,
+      assertThat(
+        assertThrows<RuntimeException> {
```

Comment:
I noticed you replaced this with assertThatThrownBy elsewhere. Just wondering why?

Reply:
Nope I'm talking rubiish! Please ignore